src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -81,6 +81,17 @@ public UniquePersonList(UniquePersonList source) {
         return Collections.unmodifiableList(internalList);
     }
 
+    /**
+     * Returns an unmodifiable java List view with sorted elements cast as immutable {@link ReadOnlyPerson}s.
+     * For use with other methods/libraries.
+     * Any changes to the internal list/elements are immediately visible in the returned list.
```

Comment:
Be mindful of coding standard for (header) comment. Here the description could be `Sorts person list in alphabetical order` with a block of `@returns an unmodifiable java List view...`